src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionView.tsx

```diff
@@ -10,38 +17,54 @@ import {
     semVerOperators,
 } from 'constants/operators';
 import { useStyles } from '../ConstraintAccordion.styles';
+import {
+    PlaygroundConstraintSchema,
+    PlaygroundRequestSchema,
+} from '../../../../hooks/api/actions/usePlayground/playground.model';
 
 interface IConstraintAccordionViewProps {
-    constraint: IConstraint;
+    constraint: IConstraint | PlaygroundConstraintSchema;
     onDelete?: () => void;
     onEdit?: () => void;
+    playgroundInput?: PlaygroundRequestSchema;
+    maxLength?: number;
+    sx?: SxProps<Theme>;
 }
 
 export const ConstraintAccordionView = ({
     constraint,
     onEdit,
     onDelete,
+    sx = undefined,
+    maxLength,
+    playgroundInput,
 }: IConstraintAccordionViewProps) => {
     const { classes: styles } = useStyles();
     const [expandable, setExpandable] = useState(true);
     const [expanded, setExpanded] = useState(false);
+    const theme = useTheme();
 
     const singleValue = oneOf(
         [...semVerOperators, ...numOperators, ...dateOperators],
         constraint.operator
     );
-
     const handleClick = () => {
         if (expandable) {
             setExpanded(!expanded);
         }
     };
+    const backgroundColor = Boolean(playgroundInput)
```

Comment:
Maybe this could use a different variable name or a comment, just to clarify what's happening.